### ISSUE TITLE
Let's make it a package/module

### ISSUE THREAD

**Issue by luroblicon** [Olivier222/Market-predictor#1]
Let's structure it as a package with API calls, methods and maybe some models.
The package could have dependencies to different AI packages, plotting capabilities, parsing capabilities etc.

This is mostly to focus on the architecture and growth of this project in the long run. Also is this going to be public or private?
Obviously as it grows it will change, but with the proper structure and organization it can grow as an open source project and multiple people can contribute to it or even fork it.

**Comment by Olivier222** [comment]
I agree with the API calls / abstraction layers. That would be awesome. I would be happy to have another contributor that can help with this API.

I am mostly focused right now on the implementation of algorithmic methodologies. My mind/intuition seems to be focused on Deep Reinforcement Learning methods.

I am open to transfer to private if it is beneficial. I have it public right now because 1) I don't think anyone will see this and 2) I am open to have input from the world at this level.
